support-frontend/window.d.ts

```diff
@@ -8,25 +9,28 @@ declare global {
 	 */
 	interface Window {
 		guardian: {
-			mdapiUrl: string;
-			serversideTests?: Participations | null;
+			amazonPaySellerId: {
+				default: string;
+				uat: string;
+			};
+			email?: string;
 			enableContributionsCampaign: boolean;
 			forceContributionsCampaign: boolean;
-			v2recaptchaPublicKey: string;
-			recaptchaEnabled?: boolean;
 			geoip?: {
 				countryCode: string;
 				stateCode?: string;
 			};
-			productPrices: ProductPrices;
+			mdapiUrl: string;
 			orderIsAGift: boolean;
-			amazonPaySellerId: {
-				default: string;
-				uat: string;
-			};
+			productPrices: ProductPrices;
+			recaptchaEnabled?: boolean;
+			serversideTests?: Participations | null;
 			stripeKeyAustralia: StripeKey;
-			stripeKeyUnitedStates: StripeKey;
 			stripeKeyDefaultCurrencies: StripeKey;
+			stripeKeyUnitedStates: StripeKey;
+			uatMode: unknown;
```

Comment:
just went to have a look - uatMode should be `true` or `false` by the looks, does unknown mean that it's sometimes missing from the page?

Reply:
`unknown` is the type-safe counterpart of any so before we used it we'd have to narrow the type. Wasn't too sure what type to go for here when initially looking at this but will update it to a boolean!